### PR TITLE
feat(task): allow to define a default task

### PR DESCRIPTION
**Pull request by joelwurtz**
Fix https://github.com/jolicode/castor/issues/539 

I also added a warning to tell user when there is multiple default, i think a warning is better in this case (instead of an exception) as we can always override default 

image

### CODE REVIEW

**Review by lyrixx — APPROVED** [review]
Super 👍🏼 

Could you add a note in the CHANGELOG

**Review by pyrech — COMMENTED** [review]
Should'nt we add a test for this feature?

File: doc/getting-started/basic-usage.md [hunk 1]
```diff
@@ -103,3 +103,20 @@ function a_very_long_function_name_that_is_very_painful_to_write(): void
...
+The `Castor\Attribute\AsTask` attribute allows you to set a default task when 
+calling `castor` without any arguments:
+
+```php
+use Castor\Attribute\AsTask;
+
+use function Castor\io;
+
+#[AsTask(name: 'bar', namespace: 'foo', default: true)]
+function a_very_long_function_name_that_is_very_painful_to_write(): void
```
[the remark this answers is not in the document]
Comment (lyrixx) [reply doc/getting-started/basic-usage.md:118]: `about` could be a cool idea.
see https://github.com/jolicode/docker-starter/blob/main/.castor/docker.php#L27-L28

**Review by pyrech — APPROVED** [review]
LGTM, thanks

**Comment by lyrixx** [comment]
@ruudk https://github.com/jolicode/castor/releases/tag/v0.20.0 :)

**Pull request merged**